src/Defs.h

```diff
@@ -114,7 +114,7 @@ extern uint32_t MPS_MAX;
 const uint32_t MPS_DEFAULT         = 10*1000;
 const uint32_t REPLY_EVERY_DEFAULT = 100;
 
-const uint32_t TEST_START_WARMUP_MSEC = 50;
+const uint32_t TEST_START_WARMUP_MSEC = 400;
```

Comment:
@igor-ivanov. I think that it is excluded. Check the set_client_timer function in the Client.cpp. Both of the TEST_START_WARMUP_MSEC and the TEST_END_COOLDOWN_MSEC are added to the total test duration to be excluded from the run results.